Assets/Scripts/Scenes/InGame/Player/Item/PlayerItem.cs

```diff
@@ -533,6 +538,19 @@ public void UseGeigerCounter(bool value)
                 _geigerCounterMove.StopCoroutine("MeasureGeigerCounter");
                 _geigerCounterMove.TurnOffGeigerCounter();
             }
+         }
+        
+        public void ActiveFireCracker(bool value)
+        {
+            _usingFireCracker.SetActive(value);
+        }
+
+        public void UseFireCracker()
+        {
+            GameObject cracker = Instantiate(_explosionFireCracker, _CrackerthrowPoint.position, _explosionFireCracker.transform.rotation);
+            cracker.GetComponent<Rigidbody>().velocity = _mainCamera.transform.forward * 10f + _mainCamera.transform.up * 2.5f;
```

Comment:
```suggestion
            cracker.GetComponent<Rigidbody>().velocity = _mainCamera.transform.forward * 10f + _mainCamera.transform.up * 2.5f;
```
_mainCameraという変数が存在しないため、起動するとsafeModeに入ります。